Seminar2/task3.py

```diff
@@ -0,0 +1,29 @@
+num1 = str(input('Введите первую десятичную дробь вида a/b: '))
+num2 = str(input('Введите вторую десятичную дробь вида a/b: '))
+num1_list = num1.split("/")
```

Comment:
Можно распаковать в 2 переменные так
```suggestion
a, b = map(int, num1.split("/"))
```